omelet-core/src/main/java/omelet/driver/DriverUtility.java

```diff
@@ -233,7 +236,60 @@ public static void selectDropDown(WebElement webElement,
 			s.selectByIndex(defaultIndex);
 		}
 	}
-
+	
+	/**
+	* select a drop down value by using partial text comparison
+	* @param element
+	* @param partialText
+	* @author nageshM
+	*
+	*/
+	public static void selectByPartialText(WebElement element, String partialText) {
+		List<WebElement> optionList = element.findElements(By.tagName("option"));
+		for (WebElement option : optionList) {
+			if (option.getText().toLowerCase().contains(partialText.toLowerCase())) 
+				option.click();
+				break;
+		}
+	}
+	
+	/***
+	 * Accept Or Dismiss Window Alert
+	 * @param driver
+	 * @param acceptOrDismiss
+	 * 
+	 * @author nageshM
+	 */		
+	public static void acceptOrDismissAlert(WebDriver driver,String acceptOrDismiss) {
+		Alert alert = driver.switchTo().alert();
+		if (acceptOrDismiss.toLowerCase().startsWith("a")) {
+			alert.accept();
+		} else if (acceptOrDismiss.toLowerCase().startsWith("d")) {
+			alert.dismiss();
+		}
+	}
+	
+	/***
+	 * switch to frame
+	 * @param driver
+	 * @param element
+	 * 
+	 * @author nageshM
+	 */
+	public static void switchToFrame(WebDriver driver, WebElement element) {
```

Comment:
Hi Nagesh 

Below methods are just one liner , so for me there is no need to add any wrapper to it.